qp_deblur/__init__.py

```diff
@@ -42,7 +42,9 @@
     'Minimum dataset-wide read threshold': ['integer', '0'],
     'Minimum per-sample read threshold': ['integer', '2'],
     'Threads per sample': ['integer', '1'],
-    'Jobs to start': ['integer', '1']
+    'Jobs to start': ['integer', '1'],
+    'Reference phylogeny for SEPP': ['choice:["Greengenes_13.8"]',
```

Comment:
This should be the path to the actual tree, perhaps worth updating?

Reply:
Looking at the code below, we could have for example: /bla/ble/bli/file and the code below can use .nwk/fasta. Thoughts?

Reply:
I was thinking about the name that is facing the user in the end and I figured "Greengenes_13.8" is more speaking than /bla/ble/bli/gg13.8/. Therefore, my code below makes an explicit mapping of the "name" to the actual files or the default files.

Easy to change, but we need to decide what the user shall see.